lib/static/new-ui/components/UiModeHintNotification/index.module.css

```diff
@@ -0,0 +1,91 @@
+@keyframes notification-appear {
+    0% {
+        opacity: 0;
+        visibility: visible;
+        filter: blur(20px);
+    }
+
+    30% {
+        opacity: 1;
+    }
+
+    80% {
+        filter: blur(0px);
+        transform: scale(1.02);
+    }
+
+    100% {
+        filter: blur(0px);
+        transform: scale(1);
+        visibility: visible;
+    }
+}
+
+@keyframes notification-disappear {
+    0% {
+        opacity: 1;
+        visibility: visible;
+    }
+
+    70% {
+        opacity: 1;
+    }
+
+    100% {
+        filter: blur(20px);
+        opacity: 0;
+        visibility: hidden;
+    }
+}
+
+.container {
+    filter: blur(0px);
+    position: fixed;
+    bottom: 6px;
+    left: 62px;
+    z-index: 999;
+    background: #6c47ff;
+    padding: 14px 14px;
+    border-radius: 10px;
+    color: rgba(255, 255, 255, .9);
+    fill: rgba(255, 255, 255, .9);
+    width: 700px;
+    display: flex;
+    gap: 8px;
+    box-shadow: 0 0 16px 0 #00000036;
+    align-items: center;
+
+    visibility: hidden;
```

Comment:
Can we remove "visibility: hidden" from here and from "notification-appear" animation?
Feels weird, when "container" class is hidden by default (even when if it is notification hint container)

Reply:
We can't, discussed this in chat